### PR TITLE
implement PurgeableBehavior

### PR DESCRIPTION
**Pull request by mjauvin**

### CODE REVIEW

File: Plugin.php [hunk 1]
```diff
@@ -54,6 +54,7 @@ public function register()
          */
         File::extend(function ($model) {
             $model->addDynamicProperty('translatable', ['title', 'description']);
+            $model->extendClassWith('October\Rain\Database\Behaviors\Purgeable');
             $model->extendClassWith('RainLab.Translate.Behaviors.TranslatableModel');
```
Comment (LukeTowers) [inline Plugin.php]: Do we need to add a `purgeable` property? Also, could you make the style of both these imports match (`\` vs `.`)? Don't really care which one, just pick one

Comment (mjauvin) [reply Plugin.php]: no, the purgeable property gets added by the Behavior and it automatically add the dynamicProperties to the purgeable property

Comment (LukeTowers) [reply Plugin.php]: Sweet

**Comment by mjauvin** [comment]
Luke, can these be merged in the wip branch so that it's easier for people to test it?

**Pull request merged**